services/ui-src/src/components/index.ts

```diff
@@ -71,8 +71,8 @@ export { HelpPage } from "./pages/Help/HelpPage";
 export { HomePage } from "./pages/Home/HomePage";
 export { NotFoundPage } from "./pages/NotFound/NotFoundPage";
 export { ProfilePage } from "./pages/Profile/ProfilePage";
-export { GetStartedPage } from "./pages/GetStarted/GetStartedPage";
-export { ReviewSubmitPage } from "./pages/ReviewSubmit/ReviewSubmitPage";
+export { McparGetStartedPage } from "./pages/McparGetStarted/McparGetStartedPage";
+export { McparReviewSubmitPage } from "./pages/McparReviewSubmit/McparReviewSubmitPage";
```

Comment:
Thought: Would it make sense to leave the folders as `GetStarted` and `ReviewSubmit` since there will be others?

Reply:
That's a good call, I hadn't considered that. Let me make the switch